qa/workunits/rbd/rbd-nbd.sh

```diff
@@ -472,6 +472,15 @@ DEV=
 rbd feature disable ${POOL}/${IMAGE} journaling
 rbd config image rm ${POOL}/${IMAGE} rbd_discard_granularity_bytes
 
+# test stuck with disabling any feature
+DEV=`_sudo rbd device --device-type nbd map ${POOL}/${IMAGE}`
+rbd feature enable ${POOL}/${IMAGE} journaling
+get_pid ${POOL}
```

Comment:
`get_pid()` logically groups with the map (or attach) command, not with anything related to image features.  I would swap this with the previous line.